update_dependants.py

```diff
@@ -112,8 +113,20 @@ def run_main() -> int:
     with open('gh_token', 'r') as f:
         gh_token = f.read().strip()
 
-    return update_dependant(cli_args.source, cli_args.platform, cli_args.version, gh_token)
+    try:
+        return update_dependant(cli_args.source, cli_args.platform, cli_args.version, gh_token)
+    except GitCommandError as e:
+        raise SanitizedGitCommandError(gh_token, e).with_traceback(e.__traceback__) from None
```

Comment:
if you use `from e` here instead of setting the `traceback` is not the same thing ?

Reply:
no, `from e` with chain exception, so the original exception will be logged as well - and we want to avoid it, we want only take the stacktrace for the ease of debugging.

Reply:
Tks for the explanation